README.md

```diff
@@ -86,16 +86,8 @@ the source tarballs, having a local copy of the git repository with
 the entire project history gives you much more insight into the
 code base.
 
-The official OpenSSL Git Repository is located at [git.openssl.org].
-There is a GitHub mirror of the repository at [github.com/openssl/openssl],
-which is updated automatically from the former on every commit.
-
 A local copy of the Git Repository can be obtained by cloning it from
-the original OpenSSL repository using
-
-    git clone git://git.openssl.org/openssl.git
-
-or from the GitHub mirror using
+from the GitHub
```

Comment:
Typo in commit message.
From is duplicated with the From on the end of previous line.

the GitHub **repository**.

Reply:
@Sashan this still needs fixing

Reply:
```suggestion
from the GitHub repository using
```

Reply:
The commit message typo and the duplicated `from` have been fixed, but `cloning it from the GitHub` isn't right: Github is a proper noun, or can be used as an adjective, so this should be changed, either just to

```
from GitHub:
```

or using Tomáš' suggestion

```
from the GitHub repository using
```